### PR TITLE
Updated all license file headers

### PR DESCRIPTION
**Pull request by sebialex**
- added new LICENSE.md file
- added new NOTICE.md file
- both are empty and should be filled in

### CODE REVIEW

**Comment by sebialex** [comment]
Hi @danielpgross, I update the files with the new header, hopefully I haven't missed one.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# TODO
```
Comment (danielpgross) [inline LICENSE.md]: Are these necessary? I think it's standard to use plain-text. We can always follow up with individuals to provide more detailed and nicely formatted information if needed.

Comment (sebialex) [inline LICENSE.md]: Do you mean just change them to .txt and leave them empty?

Comment (danielpgross) [inline LICENSE.md]: No, I mean the plain-text files [already exist](https://github.com/Gene42/phenotips-niaid/blob/master/LICENSE) [in the project](https://github.com/Gene42/phenotips-niaid/blob/master/NOTICE).

Comment (sebialex) [inline LICENSE.md]: Yeah I just saw, eh, I really don't like this stuff, two people adding the same file and I don't get a warning.

Comment (danielpgross) [inline LICENSE.md]: That's why it's important to `git pull` often 😀

**Pull request merged**